reference/types/tuple.md

```diff
@@ -14,11 +14,17 @@ This exercise asks student to create and maintain few functions which use tuples
 - Getting values from a tuple using pattern matching
 - Creating new tuples
 
+### Santa's Helper
+
+This exercise asks the student to the student to implement a pair of functions which extract values from tuples, create tuples, and work with named tuple fields.
+
 #### Implementations
 
 | Track  | Exercise                        | Changes |
 | ------ | ------------------------------- | ------- |
 | Elixir | [tuples][implementation-elixir] | None    |
+| Swift  | [tuples][implementation-swift]  | None    |
```

Comment:
Shouldn't there be two separate tables, each for each story?

Reply:
I do believe you are correct. It certainly makes more sense that way.

I'll post a correction; thanks for catching this.